src/boilerplate/contract/solidity/raw/FunctionBoilerplateGenerator.ts

```diff
@@ -42,9 +42,9 @@ class FunctionBoilerplateGenerator {
       return [
         ...(newNullifiers ? [`uint256[] ${visibility} newNullifiers`] : []),
         ...(commitmentRoot ? [`uint256 commitmentRoot`] : []),
-        ...(newCommitments ? [`uint256[] ${visibility} newCommitments`] : []),
-        ...(checkNullifiers ? [`uint256[] ${visibility} checkNullifiers`] : []),
-        ...(newCommitments || newNullifiers ? [`uint256[] ${visibility} proof`] : []),
+        ...(newCommitments ? [`uint256[] calldata newCommitments`] : []),
+        ...(checkNullifiers ? [`uint256[] calldata checkNullifiers`] : []),
+        ...(newCommitments || newNullifiers ? [`uint256[] calldata proof`] : []),
```

Comment:
```suggestion
        ...(newCommitments ? [`uint256[] ${visibility} newCommitments`] : []),
        ...(checkNullifiers ? [`uint256[] ${visibility} checkNullifiers`] : []),
        ...(newCommitments || newNullifiers ? [`uint256[] ${visibility} proof`] : []),
```

We need to keep these changes from master, so the constructor circuits works :)